README.md

```diff
@@ -40,6 +40,30 @@ Table references and indexes in a migration will be applied to the same tenant p
 Apartmentex.new_tenant(Repo, tenant)
 ```
 
+When you need to update a tenant's schema based on new migrations, you can run:
+
+```elixir
+# Runs all migrations necessary for the tenant, based on that tenant's
+`schema_migrations` table
+# Returns a tuple that is either:
+# {:ok, prefix_of_tenant, versions_migrated}
+# {:error, prefix_of_tenant, db_error_message}
+
+{status, prefix, versions_or_error} = Apartmentex.migrate_tenant(Repo, tenant)
```

Comment:
Yes, I think you're right - passing through the options to Ecto will give us the most flexibility and require us to write less code - seems like a win-win. I'll work on updating this 👍